devicetypes/smartthings/zwave-lock-without-codes.src/zwave-lock-without-codes.groovy

```diff
@@ -533,7 +533,12 @@ def refresh() {
  * @returns ret: The encapsulated command
  */
 private secure(physicalgraph.zwave.Command cmd) {
-	zwave.securityV1.securityMessageEncapsulation().encapsulate(cmd).format()
+	// Default to secure encapsulation, unless: zwaveInfo is defined and the device is secure and the command is one we should not encapsulte
+	if (!(zwaveInfo?.zw?.contains("s") && zwaveInfo.sec?.contains(String.format("%02X", cmd.commandClassId)))) {
```

Comment:
You have two different implementation of the check, maybe I am missing something, but why not go with `f (!(zwaveInfo?.zw?.contains("s") && zwaveInfo.sec?.contains(String.format("%02X", cmd.commandClassId))))` across all DTHs?